frontend/package.json

```diff
@@ -6,22 +6,23 @@
     "connected-react-router": "^6.3.1",
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.7.1",
+    "history": "^4.7.2",
```

Comment:
Joey said these should be from bootstrap https://react-bootstrap.github.io/components/overlays/#tooltips

What was wrong with these?

Reply:
I'll take a look at this and try to make it working with the bootstrap one.